src/server/views/howtos.pug

```diff
@@ -0,0 +1,204 @@
+extends includes/layout.pug
+block title
+    title  How To's - NetsBlox
+block content
+    mixin ytPlayer(url, title)
+        .panel.panel-default
+            .panel-body
+                .embed-responsive.embed-responsive-16by9
+                      iframe(class="embed-responsive-item", allowfullscreen="allowfullscreen", src= url)
+    mixin entry(question, slug, ytUrl)
+        .row.vertical-align.entry(id= slug)
+            if ytUrl
+                .col-md-8
+                    h3= question + '?'
```

Comment:
I would phrase them more like "Sending and Receiving Messages" rather than "How to send and receive messages?"